### ISSUE TITLE
Cannot load pre-trained weights

### ISSUE THREAD

**Issue by devisperessutti** [devglobalpartners/ramp-code#5]
Hello and thank you for providing this resource !

I am trying to run inference (e.g. `ramp-code/scripts/get_model_predictions.py`) using the baseline checkpoints downloaded from [RadiantEarth](https://github.com/radiantearth/model_ramp_baseline/tree/main/data/input/checkpoint.tf), but I'm getting the following error:

```
Traceback (most recent call last):
  File "ramp-code/scripts/get_model_predictions.py", line 87, in <module>
    main()
  File "ramp-code/scripts/get_model_predictions.py", line 64, in main
    model = tf.keras.models.load_model(model_dir, compile=False)
  File "/usr/local/lib/python3.8/dist-packages/keras/utils/traceback_utils.py", line 67, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/saved_model/loader_impl.py", line 105, in parse_saved_model
    raise IOError(f"Cannot parse file {path_to_pb}: {str(e)}.")
OSError: Cannot parse file b'ramp-code/ramp-ckpts.tf/saved_model.pb': Error parsing message with type 'tensorflow.SavedModel'.
```

I noticed the `assets` folder is missing from the checkpoints. Could this be the reason?

The exact command run is

```bash
python ramp-code/scripts/get_model_predictions.py -mod ramp-code/checkpoints.tf/ -idir ramp-data/PROD/aoi0/chips/ -odir ramp-data/PROD/aoi0/pred_masks/
```

I'm using the docker image locally as compiled following the excellent instructions provided.

**Comment by cjohnstonconsulting** [comment]
Hello, and thanks for submitting an issue! 

Your command uses the model directory 'ramp-code/checkpoints.tf', but the code is complaining about a protobuf file in ramp-code/ramp-ckpts.tf. On your system, is the directory with the saved model named ramp-code/checkpoints.tf, or ramp-code/ramp-ckpts.tf? Is one a softlink to the other?

**Comment by devisperessutti** [comment]
Thanks for the quick response!

The folder is `ramp-ckpts.tf` (sorry I copied the command when I earlier renamed the folder but the error relates to the correct `ramp-ckpts.tf`). The `saved_model.pb` file is in the folder as well as the `variables` folder and `keras_metadata.pb`.

Here are the commands ran

![Screenshot from 2022-12-02 09-41-02](https://user-images.githubusercontent.com/14233816/205252052-3d6bd68f-d78a-429a-a1dc-29c77529b7e4.png)

I actually then checked the sizes and realised the issue. Probably the files didn't copy correctly when I cloned the repo from radiant earth (perhaps LFS missing).

![Screenshot from 2022-12-02 09-49-56](https://user-images.githubusercontent.com/14233816/205254103-d9a118cd-eb6e-4944-b9ba-a9d28cdc24b3.png)

I downloaded the model files again and the command runs !

![Screenshot from 2022-12-02 09-55-19](https://user-images.githubusercontent.com/14233816/205254851-cb6fadfc-175e-4f37-9d41-eddf770a486f.png)


I need to debug now why predictions are not written but I will open another issue if needed.

Thanks!
